internal/pkg/manifest/svc.go

```diff
@@ -300,12 +314,48 @@ func (a *Count) UnmarshalYAML(unmarshal func(interface{}) error) error {
 // Autoscaling represents the configurable options for Auto Scaling.
 type Autoscaling struct {
 	Range        Range          `yaml:"range"`
-	CPU          *string        `yaml:"cpu"`
-	Memory       *string        `yaml:"memory"`
+	CPU          *Percentage    `yaml:"cpu"`
+	Memory       *Percentage    `yaml:"memory"`
```

Comment:
What do you think about keeping these dumber like this?
```suggestion
	CPU          *int    `yaml:"cpu_percentage"`
	Memory       *int    `yaml:"memory_percentage"`
```
so that it reads:
```yaml
autoscaling:
  range: 1-100
  cpu_percentage: 70
```